homeassistant/components/nanogrid_air/config_flow.py

```diff
@@ -0,0 +1,65 @@
+"""Config flow for Nanogrid Air integration."""
+
+from __future__ import annotations
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_URL
+
+from .api import fetch_mac, get_ip
+from .const import DOMAIN
+
+API_DEFAULT = "http://ctek-ng-air.local/"
```

Comment:
should be "http://ctek-ng-air.local/meter/"

Reply:
fixed